test/e2e-tests.sh

```diff
@@ -146,8 +131,14 @@ wait_until_service_has_external_ip istio-system knative-ingressgateway || fail_t
 
 # Run the tests
 
-result=0
-run_e2e_tests conformance pizzaplanet || result=1
-run_e2e_tests e2e noodleburg || result=1
-[[ ${result} -ne 0 ]] && exit 1 # run_e2e_tests already dumps state
+header "Running tests"
+kubectl create namespace serving-tests
+kubectl label namespace serving-tests istio-injection=enabled --overwrite
+options=""
+(( EMIT_METRICS )) && options="-emitmetrics"
+report_go_test -v -tags=e2e -count=1 \
```

Comment:
this doesn't run go test in parallel mode. So by "concurrent", it only means that both e2e and conformance are run in a single go test cmd. Right?

Reply:
By "concurrently"  I do mean concurrently goroutines. Further paralellization can be achieved through `t.Parallel()`, which is being explored by Srinivas.

/hold cancel